dbms/src/Server/CMakeLists.txt

```diff
@@ -125,7 +125,9 @@ if (OS_LINUX)
                 DESTINATION ".")
     endif()
 
-    install (SCRIPT ${TiFlash_SOURCE_DIR}/cmake/tiflash_linux_post_install.cmake COMPONENT tiflash-release)
+    if (NOT (CMAKE_BUILD_TYPE_UC STREQUAL "DEBUG" OR SAN_DEBUG))
+        install (SCRIPT ${TiFlash_SOURCE_DIR}/cmake/tiflash_linux_post_install.cmake COMPONENT tiflash-release)
+    endif()
```

Comment:
I think we can keep this under debug mode as the tiflash binary is too large under debug mode. And reducing the binary size can speedup our CI.

Reply:
Makes sense for me.